docs/source/index.rst

```diff
@@ -55,7 +55,7 @@ The current list of known implementations can be found as follows
     known_implementations
 
 These are only imported on request, which may fail if a required dependency is missing. The dictionary
-``fsspec.registry`` contains all imported implementations, and can be mutated by user code, if necessary.
+:py:class:`fsspec.registry.known_implementations` contains all imported implementations, and can be mutated by user code, if necessary.
```

Comment:
Actually, on second reading, this is wrong. `fsspec.registry` is indeed the set of classes that have been used so far: the values are python class objects. `known_implementations` is a dictionary where the values are dictionaries of import details, location of the class and reasonable error messages - it serves as the source of the classes for `registry` via `fsspec.get_filesystem_class`, but you could edit either one at runtime.